analyses/cms-open-data-ttbar/analysis.py

```diff
@@ -270,8 +253,8 @@ def book_histos(
 
     # not strict condition is used because the same selection cut is applied in the reference implementation
     # https://github.com/iris-hep/analysis-grand-challenge/blob/main/analyses/cms-open-data-ttbar/ttbar_analysis_pipeline.py#L254
-    df4j1b = df.Filter("Sum(Jet_btagCSVV2[Jet_pt_mask]>=0.5)==1")\
-               .Define("HT", "Sum(Jet_pt[Jet_pt_mask])")
+    df4j1b = df.Filter("Sum(Jet_btagCSVV2_masked>=0.5)==1")\
```

Comment:
I know, I know, I'm nitpicking, but since we are here... :)

```suggestion
    df4j1b = df.Filter("Sum(Jet_btagCSVV2_masked >= 0.5) == 1")\
```